bindings/java/src/test/com/apple/foundationdb/test/TuplePerformanceTest.java

```diff
@@ -169,7 +173,13 @@ public void run() {
 		System.out.println("Warming up test...");
 		for(int i = 0; i < ignoreIterations; i++) {
 			int length = r.nextInt(20);
-			createTuple(length).pack();
+			Tuple t = createTuple(length);
+			try {
+				t.pack();
+			}
+			catch(IllegalArgumentException e) {
+				System.out.println("ah!");
```

Comment:
Probably not a huge deal since this is a test, but a better error message may end up being helpful at some point in the future.

Reply:
I added this basically so that I had a place to put my debugger. I've reverted so that it just throws the raised exception.